packages/lesswrong/components/posts/dialogues/DialogueSubmit.tsx

```diff
@@ -107,6 +118,33 @@ const DialogueSubmit = ({
         >
           {saveDraftLabel}
         </Button>
+        {userShortfeed && <Button
+          className={classNames(classes.formButton)}
+          disabled={loading || !!error}
+          onClick={async e => {
+            e.preventDefault()
+            
+            // So this property exists. But the typings don't agree. For now, #AnyBecauseHard
+            // @ts-ignore
+            const shortformString = editors[`${document._id}-contents`].getData()
+            const shortformContents = {originalContents: {type: "ckEditorMarkup", data: shortformString}}
+
+            const response = await createShortform({
+              data: {
+                // When Ollie Etherington wrote ff6296e he ts-ignore'd the annoying error.
+                // And he's definitely a real developer. So I'm going to do the same.
+                // @ts-ignore
```

Comment:
If the issue is that `shortformContents` isn't of the type expected by the function spat out by `useCreate` for that field, then I think it's probably clearer to type-cast the field itself (`shortformContents as DbComment['contents']`)